### ISSUE TITLE
Cookies with Redirects

### ISSUE THREAD

**Issue by PHPDev-CA** [EFForg/privacybadgerfirefox#649]
There appears to be a significant oversight in how PrivacyBadger functions whereby any cookie that is set and then immediately followed by a redirect never shows up on the cookie list.  This creates issues with OpenID style services where a unique ID is being passed between sites quite legitimately.  If it's a single redirect disabling PB on the initiating site is enough.  However, if there is a 2nd redirect PB must be disabled completely to be able to sign in to certain sites (example rewards.xbox.com which logs in via 2 sites - eprize.com and live.com; both cookies must be present or it immediately logs you out).

**Comment by cooperq** [comment]
You should be able to whitelist the individual domains needed for oauth to work in the privacy badger settings page at any time. 
Do you have a suggestion about what we should do to allow oauth logins that have a redirect?

**Comment by PHPDev-CA** [comment]
I gave up on PrivacyBadger.  It was breaking more than 50% of the sites I visit due to non-tracking cookies, the interface didn't provide enough useful information on what it was blocking to be able to fix it easily (basically a guessing game as to what might unbreak the site).  Mostly CSS/login scripts would break.  After months of trying I came to the conclusion that the algorithmic approach simply will not work and that a blacklist ala AdBlock is the only realistic solution.
